vignettes/betareg.Rmd

```diff
@@ -23,36 +23,40 @@ params:
 
 # Introduction
 
-This vignette explains how to model continuous outcomes on the open unit interval
-using the `stan_betareg` function in the __rstanarm__ package.
+This vignette explains how to model continuous outcomes on the open unit
+interval using the `stan_betareg` function in the __rstanarm__ package.
 
 ```{r, child="children/four_steps.txt"}
 ```
 
-Steps 3 and 4 are covered in more depth by the vignette entitled ["How to Use the
-__rstanarm__ Package"](rstanarm.html). This vignette focuses on Step 1 when the likelihood is
-the product of beta distributions.
+Steps 3 and 4 are covered in more depth by the vignette entitled ["How to Use
+the __rstanarm__ Package"](rstanarm.html). This vignette focuses on Step 1 when
+the likelihood is the product of beta distributions.
 
 # Likelihood
 
 Beta regression uses the beta distribution as the likelihood for the data,
 $$
 f(y_i | \alpha,\beta) = \frac{x_i^{(\alpha-1)}(1-x_i)^{(\beta-1)}}{B(\alpha,\beta)}
 $$
-where $B(\cdot)$ is the beta function. The shape parameters for the distribution are $\alpha$ and $\beta$. The shape parameters enter
-into the model according to the following transformations.
+where $B(\cdot)$ is the beta function. The shape parameters for the distribution
+are $\alpha$ and $\beta$. The shape parameters enter into the model according to
+the following transformations.
 $$
 \alpha = \mu\cdot\phi \\
 \beta = (1-\mu)\cdot\phi
 $$
 
-Where $g_1(\mu) = \mathbf{X}\boldsymbol{\eta}$ where $g_1(\cdot)$ is some link function, $X$ is a $n\times K$ dimensional matrix of
-predictors, and $\eta$ is a $K$ dimensional vector of parameters associated with each predictor. 
+Where $g_1(\mu) = \mathbf{X}\boldsymbol{\eta}$ where $g_1(\cdot)$ is some link
```

Comment:
I think I just read  "Where g_1(mu) ..." as referring to a previous usage of g_1 above when there isn't one. It's definitely not wrong as is, but I think it would be good to write out the likelihood using the g_1(mu) and g_2(phi) terms somewhere. Maybe I'm missing it but I don't think that's written out anywhere. No big deal though.